tests/OmniSharp.MSBuild.Tests/ProjectFileInfoTests.cs

```diff
@@ -104,5 +104,25 @@ public async Task NetStandardAndNetCoreApp_has_correct_property_values()
                 Assert.Equal("AnyCPU", projectFileInfo.Platform);
             }
         }
+
+        [Fact]
+        public async Task ExternAlias()
```

Comment:
this test is failing at the moment

Reply:
forget to add test assets to build.json, now fixed.